requirements.txt

```diff
@@ -9,3 +9,4 @@ gevent>=1.0
 greenlet>=0.3.2
 # This is a sub-dependency of ansible, Its not automatically pulled. So explictly pulling this.
 setuptools
+psutil
```

Comment:
@mbukatov Please note this additional requirement for https://github.com/Tendrl/node-agent/pull/236